package.json

```diff
@@ -0,0 +1,46 @@
+
+
+
+{
+  "name": "15-bearer_auth",
+  "version": "1.0.0",
+  "description": "msg: invalid signature\r name: JsonWebTokenError",
+  "main": "gulpfile.js",
+  "directories": {
+    "test": "test"
+  },
+  "dependencies": {
+    "bcryptjs": "^2.4.0",
+    "bluebird": "^3.4.7",
+    "body-parser": "^1.15.2",
+    "cors": "^2.8.1",
+    "crypto": "0.0.3",
```

Comment:
You don't need to list `crypto` in your package.json because it is [built into](https://nodejs.org/api/crypto.html) node.